### DISCUSSION TITLE
Wallpapers Source

### DISCUSSION THREAD

**Discussion by PrettyKawaii** [tanujnotes/Olauncher#520] [General]
Hi there! 
I really enjoy Olauncher and I've been wondering if I could download the daily wallpapers. They are simply gorgeous!
Thank you! :)

**Comment by tanujnotes** [comment]
If you tap 'Daily new wallpaper' text, you can download them.

**Reply by PrettyKawaii** [reply]
Oh, thank you! It wasn't obvious to me :D

**Comment by Originalwhite** [comment]
> If you tap 'Daily new wallpaper' text, you can download them.

Is it possible to find a wallpaper from a few days ago? It only brings me to the current wallpaper. I was served some really cool ones that i would like to go back to some time.

**Reply by CreativeCodeCat** [reply]
https://gist.githubusercontent.com/tanujnotes/85e2d0343ace71e76615ac346fbff82b/raw

here it the json for them all.

just search for 6_12 if you want the one from 12th of June for example

**Reply by Heike-HS** [reply]
You can change the date of your phone. This brings you to the wallpaper for that day.

**Comment by ghost** [comment]
Using @CreativeCodeCat 's link, 
Here is html files to view them in browser. This is not professionally made, And mainly made for phone browser.
The image resolution are limited to 960p at the shorter end (if code works correctly).

[Wallpaper Olauncher.html](https://github.com/user-attachments/files/30424959/Wallpaper.Olauncher.html)
[Olauncher Dark.html](https://github.com/user-attachments/files/30424964/Olauncher.Dark.html)
[Olauncher Light.html](https://github.com/user-attachments/files/30424965/Olauncher.Light.html)

**Comment by Heike-HS** [comment]
You can of course chose whatever picture file you have in your smartphones "harddisk". Have a look directly an unsplash.com and download your own favorites.
